docs/Demo.md

```diff
@@ -1,447 +1,409 @@
-    
-        helm install pelorus charts/pelorus --namespace pelorus
-
-* Wait for the Pelorus install to complete
-
-        $ oc get pods -n pelorus
-        NAME                                                   READY     STATUS      RESTARTS      AGE
-        deploytime-exporter-1-deploy                           0/1       Completed   0             93s
-        deploytime-exporter-1-rwk5l                            1/1       Running     0             90s
-        grafana-deployment-55f77ccc8f-d7m92                    2/2       Running     0             84s
-        grafana-operator-controller-manager-7678cc5c7c-spvls   2/2       Running     0             4m5s
-        prometheus-operator-559d659944-fvsjg                   1/1       Running     0             3m53s
-        prometheus-prometheus-pelorus-0                        3/3       Running     1 (89s ago)   93s
-        prometheus-prometheus-pelorus-1                        3/3       Running     1 (89s ago)   93s
+> **Note:** More information about the four key DORA metrics can be found in the [Software Delivery Performance](https://GitHub.com/konveyor/konveyor.GitHub.io/blob/main/content/Pelorus/SoftwareDeliveryPerformance.md) section.
 
-* Finally check the install
-
-        oc get all -n pelorus
-    
-
-### Pelorus configuration
-
-* Make a copy of [the values.yaml file in the pelorus repo](https://github.com/konveyor/pelorus/blob/master/charts/pelorus/values.yaml) ([raw link for curl-ing](https://raw.githubusercontent.com/konveyor/pelorus/master/charts/pelorus/values.yaml)) and save it to /var/tmp/values.yaml
-
-        cp charts/pelorus/values.yaml /var/tmp/
-
-* Update the "exporters" section of the /var/tmp/values.yaml to match the following example:
-
-        exporters:
-          instances:
-          - app_name: deploytime-exporter
-            exporter_type: deploytime
-            extraEnv:
-            - name: LOG_LEVEL
-              value: DEBUG
-            - name: NAMESPACES
-              value: mongo-persistent
-          - app_name: committime-exporter
-            exporter_type: committime
-            extraEnv:
-            - name: LOG_LEVEL
-              value: DEBUG
-            - name: NAMESPACES
-              value: mongo-persistent
-
->**Note:** [Documentation regarding values.yaml can be found on our readthedocs page.](https://pelorus.readthedocs.io/en/latest/Configuration/)
-
-* Apply the updated values for Pelorus by executing:
-
-        helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
-
-> **Note:** Please pause to allow the committime exporter pod to be deployed.
-
-Pelorus should now be installed and configured to measure the todolist sample app. We'll have to deploy the sample application to view measurements from Pelorus's Grafana dashboard.
-
-## Todolist sample application install steps
-
-* git clone the forked copy of konveyor/mig-demo-apps
+## Installing Pelorus
+There are two steps to installing Pelorus:
```

Comment:
Probably not the intent, to test locally.
```
cd pelorus
make dev-env
source .venv/bin/active
mkdocs build
```
* it will return a path to local filesystem where the docs were built
* navigate to url ->  file:///path/<snip>/site

Reply:
I haven't had the opportunity to install the env and test locally.  I asked Kevin to help me with that.